### PR TITLE
refactor: leave type

### PR DESCRIPTION
**Pull request by sammish**
This feature will help disable and enable each leave type name in the leave type.

### CODE REVIEW

File: erpnext/hr/doctype/leave_type/leave_type.json [hunk 1]
```diff
@@ -12,15 +12,13 @@
...
   "is_carry_forward",
   "is_lwp",
   "is_optional_leave",
   "allow_negative",
   "include_holiday",
   "is_compensatory",
-  "carry_forward_section",
-  "maximum_carry_forwarded_leaves",
-  "expire_carry_forwarded_leaves_after_days",
   "encashment",
```
Comment (Mangesh-Khairnar) [inline erpnext/hr/doctype/leave_type/leave_type.json:22]: Add these fields back

File: erpnext/hr/doctype/leave_type/leave_type.json [hunk 2]
```diff
@@ -168,15 +153,15 @@
    "options": "0.5\n1.0"
   },
   {
-   "depends_on": "is_carry_forward",
-   "fieldname": "maximum_carry_forwarded_leaves",
-   "fieldtype": "Float",
-   "label": "Maximum Carry Forwarded Leaves"
+   "default": "0",
```
Comment (Mangesh-Khairnar) [inline erpnext/hr/doctype/leave_type/leave_type.json:156]: Same as above

**Comment by Mangesh-Khairnar** [comment]
You need to add related functionality and describe the reason behind the proposed feature. I am closing this PR for now. Since it is unusable in its current form.

**Comment by sammish** [comment]
@Mangesh-Khairnar okay let me check. thank you fro your feedback.

**Pull request closed without merging**